winui/AvatarView/Overview.md

```diff
@@ -0,0 +1,21 @@
+---
+layout: post
+title: Overview of WinUI AvatarView control | Syncfusion
+description: Learn all about the introduction of Syncfusion WinUI AvatarView Control (SfAvatarView) with essential features and more here.
+platform: WinUI
+control: SfAvatarView
+documentation: ug
+---
+
+# WinUI Avatar View (SfAvatarView) Overview
+
+The AvatarView control provides a graphical representation of user image that allows you to customize the view by adding image, background color, icon, text, etc.
+
+![AvatarView control overview in WinUI](AvatarView_images/winui_avatarview_overview.png)
+
+## Key features
+
+* Supports for adding image, initials and group.
```

Comment:
@MadhumithanJayapathi 

* `Content type`: Supports for adding image, initials and 25 predefined avatars.
* `Group view`: Add maximum three custom images or initials in a single view.
* `Visual styles`: Supports different types of visual styles such as circle, square and custom. 
* `Customization`: Customizes the Height, Width, BorderBrush, BorderThickness, Background, CornerRadius and Font.

Reply:
@Manivannan-E Changed the key features as given